kubejs/server_scripts/base/entity_events/spawned/custom_rituals.js

```diff
@@ -383,6 +384,7 @@ EntityEvents.spawned((event) => {
 
                 delay = start_delay + 0.5 * block.pos[1];
                 event.server.scheduleInTicks(delay, (schedule) => {
+                // Note to Motoko: Due to pending removal of Blue_Skies, this require a change I can't do myself ^^"
```

Comment:
@MuteTiefling heads up 😄

Reply:
I think Motoko already fixed that 😅 Need to check, if so, I'm gonna remove the notes :D